src/Inventory/Conf.php

```diff
@@ -1179,6 +1195,7 @@ public static function getDefaults(): array
             'stale_agents_delay'             => 0,
             'stale_agents_action'            => exportArrayToDB([0]),
             'stale_agents_status'            => 0,
+            'previous_stale_agents_status'   => exportArrayToDB([0]),
```

Comment:
Maybe `stale_agents_status_condition` would be more understandable.

At first, I thought you was trying to introduce some kind of history on the `stale_agents_status` values.

Reply:
fixed